[PLAYER-13] adding showAdCountDown parameter to show/hide remaining time for ads

ksunika opened the pull request:
should work with https://github.com/ooyala/skin-config/pull/157

pilievOoyala commented on js/components/adPanel.js:
```diff
@@ -74,15 +74,17 @@ var AdPanel = React.createClass({
+      } else {
+        remainingTime = parseInt(this.props.adVideoDuration - this.props.currentPlayhead)
+      }
+      remainingTime = Utils.formatSeconds(Math.max(0, remainingTime));
```
we should probably have a check to make sure we have a finite number. Then we can avoid possible showing NaN on the screen.

pilievOoyala replied:
Also, add a debug log statement if the data is bad.

ksunika commented:
@pilievOoyala, I addressed your comments. For some reason I can't point the config file to the one used in the corresponding pull request in skin config. If I try, I get conflicts. Not sure how to address that.

mistermendez commented:
@ksunika I had to sync with latest config, then merge master into this branch.

Good to go.

pilievOoyala commented:
Looks good. Merging!
